### PR TITLE
Add RDM day blurb

### PR DESCRIPTION
**Pull request by joshmoore**
Replace the TBD for those who read over the weekend.

### CODE REVIEW

**Comment by joshmoore** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2021/day2/

Screen Shot 2021-04-23 at 12 44 09

File: _data/ome2021.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
       "alternateName": "Research Data Management",
       "url": "/events/ome-community-meeting-2021/day2/",
       "eventAttendanceMode": "https://schema.org/OnlineEventAttendanceMode",
-      "description": "TBD",
+      "description": "The Research Data Management (RDM) day aims to cover (and perhaps answer) the gamut of pre-publication struggles, from structuring FAIR meta-/data through discovering insights and sharing those with others, all in preparation for day 3 -- public resources.",
```
Comment (pwalczysko) [inline _data/ome2021.json:116]: 1. gamut ? (still need to check what that means)
2. ``...discovering insights....`` insights are usually "gained" or somesuch ?

Not stuck on the suggestions above, any simplification of the complexity of the sentence would do imho.

Comment (joshmoore) [reply _data/ome2021.json]: Pushed

**Comment by pwalczysko** [comment]
LGTM

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**